packages/google_maps_flutter/README.md

```diff
@@ -22,3 +22,14 @@ dependencies:
   google_maps_flutter_tizen: ^0.0.1
 ```
 
+In addition, you need a Maps JavaScript API Key to use this plugin. Please refer to the site below to get the API key.
+```
+https://developers.google.com/maps/documentation/javascript/get-api-key
```

Comment:
Consider putting this as a link. Currently developer has to select text, copy link and manually paste it into address bar. Clicking into link would be easier.

Reply:
Thank you~